include/mbedtls/config.h

```diff
@@ -2254,6 +2254,20 @@
  */
 #define MBEDTLS_ECP_C
 
+/**
+ * \def MBEDTLS_USE_UECC
+ *
+ * Enable the tinycrypt ECC library.
+ *
+ * Module:  tinycrypt/ecc.c
+ *          tinycrypt/ecc_dh.c
+ *          tinycrypt/ecc_dsa.c
+ *
+ * This module provides alternative ECC handling functions replacing
+ * native MBEDTLS ECP module.
+ */
+//#define MBEDTLS_USE_UECC
```

Comment:
Whereas everywhere else we refer to 'tinycrypt', here in the switch to enable the feature, we're calling it 'uECC'. Better to call it ` MBEDTLS_USE_TINYCRYPT`. This doesn't have to be fixed in this PR, but please fix it in the next follow-up PR.

Reply:
See https://github.com/ARMmbed/mbedtls-restricted/pull/565